### DISCUSSION TITLE
MCS%40UVa/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#287] [Announcements]
# MCS%40UVa/

None

https://opencs.app/MCS%40UVa/

**Comment by Perrkkey** [comment]
24fall哀鸿遍野，建议重新评估难度，预计A-或者A

**Comment by jimlue530** [comment]
25Fall 非常難申請在A-或以上，好處是不用申請費及托福GRE官方送分費，只要交unofficial成績，人人皆可抽獎
